ansible_collections/arista/avd/roles/eos_designs/templates/facts/switch/switch.j2

```diff
@@ -397,3 +397,37 @@ switch:
 {# switch.vlans #}
   vlans: {{ leaf.vlans | unique }}
 {% endif %}
+
+{# EVPN multihoming configuration on VTEPs #}
+{# switch.link_tracking #}
+{% if switch_data.combined.link_tracking.enabled is arista.avd.defined(true) %}
+{%     if switch_data.combined.link_tracking.groups is arista.avd.defined %}
+  link_tracking_groups:
+{#         Do not sort the group as the first group will be the default group for the end-points #}
+{%         for lt_group in switch_data.combined.link_tracking.groups %}
+{%             if lt_group.name is arista.avd.defined %}
+    - name: {{ lt_group.name }}
+      recovery_delay: {{ lt_group.recovery_delay | arista.avd.default(500) }}
```

Comment:
I think we should default to 0 / no delay. If this is seconds, it will have severe impacts on the network.

Reply:
No, recovery_delay is required as far as I understand. 500 was the value used in document. We are allowing the BGP to come up and then LAGs to work.
![image](https://user-images.githubusercontent.com/88734745/152162107-7ec02c37-2f4c-494b-b8ce-bb32a3a24863.png)

Reply:
Even for a regular use case, I think there should be a delay to allow for route convergence unless if static routing is used.

Reply:
For l3 devices (check switch.underlay_router) I think the default should be 300 to match the default mlag reload delay. For pure l2 devices where we just have a port-channel uplink, IMO the delay should be minimum. Maybe 10 seconds. Just for spanning-tree to settle.